### PR TITLE
fix: carbon gas fee estimation

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

File: src/txs/wasm/StoreCodeForm.tsx [hunk 1]
```diff
@@ -46,6 +50,7 @@ const StoreCodeForm = ({ chainID }: { chainID: string }) => {
   const tx = {
     estimationTxValues,
     createTx,
+    decimals: readNativeDenom(network[chainID].baseAsset ?? "")?.decimals ?? 6,
```
Comment (restorenode) [inline src/txs/wasm/StoreCodeForm.tsx]: Why aren't these edge case protectors on the other lines of code where the decimals are evaluated?

Comment (ericHgorski) [reply src/txs/wasm/StoreCodeForm.tsx]: meaning other transaction forms?

File: src/txs/stake/QuickStakeForm.tsx [hunk 2]
```diff
@@ -139,7 +139,7 @@ const QuickStakeForm = (props: Props) => {
 
   const token = action === QuickStakeAction.DELEGATE ? denom : ""
   const tx = {
-    decimals: readNativeDenom(token)?.decimals,
+    decimals: readNativeDenom(network[chainID].baseAsset).decimals,
```
Comment (restorenode) [inline src/txs/stake/QuickStakeForm.tsx]: Like here.

Comment (ericHgorski) [reply src/txs/stake/QuickStakeForm.tsx]: Ok removed them. Youre right, they are unnecessary, thanks!

Comment (restorenode) [inline src/txs/stake/QuickStakeForm.tsx]: Also, why was this updated from token, but line `183` in `StakeForm.tsx` was left alone?

**Comment by ericHgorski** [comment]
duplicate #485

**Pull request closed without merging**